playground/nextjs/package.json

```diff
@@ -15,8 +15,9 @@
         "@types/react-dom": "18.0.10",
         "eslint": "8.34.0",
         "eslint-config-next": "13.1.6",
+        "mapbox-gl": "^3.1.2",
         "next": "13.5.6",
-        "posthog-js": "^1.88.1",
```

Comment:
Not for this PR but I think our examples should depend on the folder so it's always using the latest/current version instead of the pinned version, you might try to test a version that isn't published yet, its not gonna work.